pwndbg/commands/__init__.py

```diff
@@ -153,7 +153,7 @@ def invoke(self, argument: str, from_tty: bool) -> None:
         except SystemExit:
             # Raised when the usage is printed by an ArgparsedCommand
             return
-        except (TypeError, gdb.error):
+        except (TypeError, pwndbg.dbg_mod.Error):
```

Comment:
Random comment: is there any reason we name it `dbg_mod` instead of just `dbg`?

Reply:
I wanted accessing the main debugger object to be concise, so I borrowed the `<name>/<name>_mod` idiom that Pwndbg uses for `pwndbg.gdblib.arch` and `pwndbg.gdblib.config` (when that was still a thing), where `<name>` is the main object of that module, and `<name>_mod` is the module itself.